### PR TITLE
Homepage animation

### PR DESCRIPTION
**Pull request by danrot**
This PR adds the homepage animation with the clouds and the various backgrounds.

### CODE REVIEW

File: components/Navigation.js [hunk 1]
```diff
@@ -100,6 +100,7 @@ export default class Navigation extends React.PureComponent {
             <List>
                 <Lines color="rgba(0, 0, 0, 0.07)" />
                 <ul>
+                    <li><a href="https://blog.agent.sh">Blog</a></li>
```
Comment (tomsvogel) [inline components/Navigation.js:103]: target="_blank" ?

File: pages/index.js [hunk 2]
```diff
@@ -1,10 +1,324 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
 import Page from '../components/Page';
```
Comment (tomsvogel) [inline pages/index.js]: Large file, would make sense to split up

File: components/Navigation.js (hunk 1 above)
Comment (danrot) [reply components/Navigation.js:103]: As a user I am not a big fan of websites which automatically open new windows, my tabs are already chaotic enough 😄 But if you think it makes sense I can still add it.

**Comment by danrot** [comment]
@tomsvogel Would you factor out even more? The rest of the index page is influenced by the content of the page itself. Extracting the `MountainSlide` and `CitySlide` doesn't make a lot sense to me, because the exact same slides are not going to be reused somewhere else.

**Pull request merged**